cfgov/v1/models/ref.py

```diff
@@ -1,21 +1,110 @@
-from wagtail.wagtailcore import blocks
-
-choices = [
-    ('info for consumers', 'Info for consumers'),
-    ('at the cfpb', 'At the CFPB'),
-    ('data, research & reports', 'Data, research & reports'),
-    ('policy & compliance', 'Policy & Compliance'),
-    ('speech', 'Speech'),
-    ('press release', 'Press Release'),
-    ('op-ed', 'Op-ed'),
-    ('testimony', 'Testimony'),
-    ('cfpb_report', 'CFPB Report'),
+page_types = [
```

Comment:
So when you're creating the BrowseFilterablePage, there are certain categories that you want associated with it. They are grouped by these page types. If we used the categories instead, then we'd force the content editor to add each category individually.